### ISSUE TITLE
use the output of blazer to solve my model

### ISSUE THREAD

**Issue by mariusmjp** [IRIS-Solutions-Team/IRIS-Toolbox#280]
Hi Jaromir

Is it possible to pass as argument the output of blazer to solve my model. If this is the case have you any tutorial or example of this.

Thanks

**Comment by jaromir-benes** [comment]
Hi - I'm not sure I understand correctly what exactly you are looking
for... Do you want to have access to the results of the blazer, i.e. the
ordering of equations and variables into blocks?

**Comment by mariusmjp** [comment]
Yes exactly for making a short history I want to have access to the results of the blazer . Try to re-order the equations and variables into the block by myself and pass the result to the model for solving.

**Comment by jaromir-benes** [comment]
Hi

you can access the blazer results using either of the following two ways:

* when running the steady(...) function to calculate the steady state of
the model, use the option "saveAs" and specify a file name under which the
blazer results describing the sequential ordering of equations in blocks
are saved.

* run separately the blazer(...) function:

[variables, equations] = blazer(m, ...)

giving you the results either in the two output arguments (those cell
arrays with individual blocks, containing the variables solved for in each
block and the equations in each block), or again by using a "saveAs" option
to get the results in a text file.

There is though no way of somehow modifying the results and feed them back
to the steady() function - this would unfortunately entail a very complex
check that the user hasn't created any inconsistencies, and would horribly
costly to implement :))

Hope this helps,
Jaromir
